medallion/backends/mongodb_backend.py

```diff
@@ -467,3 +495,66 @@ def get_object_versions(self, api_root, collection_id, object_id, filter_args, a
         manifests_found = list(map(lambda x: datetime_to_string_stix(float_to_datetime(x["version"])), manifests_found))
         next_id, more = self._update_record(next_id, count)
         return create_resource("versions", manifests_found, more, next_id), headers
+
+    def load_data_from_file(self, filename):
+        if isinstance(filename, string_types):
+            with io.open(filename, "r", encoding="utf-8") as infile:
+                self.json_data = json.load(infile)
+        else:
+            self.json_data = json.load(filename)
```

Comment:
Probably need some error handling for bad data or file not found, etc.

Reply:
wrapped to raise InitializationError